models/lifi/fantom/lifi_v2_fantom_trades.sql

```diff
@@ -26,27 +27,27 @@ dexs as (
     {% for trade_tables in trade_event_tables %}
         SELECT 
             evt_block_time as block_time,
-            '' as maker, 
+            CAST(NULL as VARBINARY) as maker, 
             toAmount as token_bought_amount_raw,
             fromAmount as token_sold_amount_raw,
             CAST(NULL as double) as amount_usd,
             CASE 
-                WHEN toAssetId IN ('0', 'O', '0x0000000000000000000000000000000000000000')
-                THEN '0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83' -- wftm 
+                WHEN toAssetId IN (0x, 0x0000000000000000000000000000000000000000)
+                THEN 0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83 -- wftm 
                 ELSE toAssetId
             END as token_bought_address,
             CASE 
-                WHEN fromAssetId IN ('0', 'O', '0x0000000000000000000000000000000000000000')
-                THEN '0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83' -- wftm 
+                WHEN fromAssetId IN (0x, 0x0000000000000000000000000000000000000000)
+                THEN 0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83 -- wftm 
                 ELSE fromAssetId
             END as token_sold_address,
             contract_address as project_contract_address,
             evt_tx_hash as tx_hash, 
-            CAST(ARRAY() as array<bigint>) AS trace_address,
+            ARRAY[-1] AS trace_address,
```

Comment:
This PR uses `ARRAY[-1]` for `trace_address` field.

Reply:
we will need to track this usage across all models, as they will eventually union downstream into `dex_aggregator.trades`

we can always revisit and fix when that final PR is built, but for us all to consider